Pod/Classes/ViewControllers/ARMapComponentViewController.m

```diff
@@ -0,0 +1,15 @@
+#import "ARMapComponentViewController.h"
+
+@implementation ARMapComponentViewController
+
+- (instancetype)init
```

Comment:
Will need to use `appProperties` (https://facebook.github.io/react-native/docs/communication-ios) to set location onto map from native User location module